src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/SafeMsQuicConfigurationHandle.cs

```diff
@@ -56,7 +56,7 @@ public static SafeMsQuicConfigurationHandle Create(QuicClientConnectionOptions o
             {
                 flags |= QUIC_CREDENTIAL_FLAGS.USE_SUPPLIED_CREDENTIALS;
             }
-            return Create(options, flags, certificate: certificate, certificateContext: null, options.ClientAuthenticationOptions?.ApplicationProtocols, options.ClientAuthenticationOptions?.CipherSuitesPolicy);
+            return Create(false, options, flags, certificate: certificate, certificateContext: null, options.ClientAuthenticationOptions?.ApplicationProtocols, options.ClientAuthenticationOptions?.CipherSuitesPolicy);
```

Comment:
I don't think you need new `bool`. Client will have QUIC_CREDENTIAL_FLAGS.CLIENT set and server not.